front/src/store/index.js

```diff
@@ -25,7 +26,16 @@ export default new Vuex.Store({
     },
     CLEAR_USER_DATA (state) {
       localStorage.removeItem('user');
+      delete axios.defaults.headers.common.Authorization;
       location.reload();
```

Comment:
this is bad practice to SPA, to forcebly reload pages

Reply:
it's not my code) i edit only 29th

Reply:
I understand that, but it was before me)